paddle/fluid/framework/parallel_executor.cc

```diff
@@ -313,6 +327,9 @@ ir::Graph *ParallelExecutorPrivate::ApplyMemoryOptimizePass(ir::Graph *graph) {
     eager_deletion_pass->SetNotOwned(ir::kAllPlaces, &places_);
     graph = eager_deletion_pass->Apply(graph);
     VLOG(10) << "EagerDeletionPass Applied";
+    LOG(INFO) << "Garbage collection strategy is enabled, when "
+              << "FLAGS_eager_delete_tensor_gb="
+              << GetEagerDeletionThreshold() / (static_cast<int64_t>(1) << 30);
```

Comment:
Can we write like this?
<< (GetEagerDeletionThreshold() >> 30);

Reply:
Done.